Registration/RegisterWithAzure.psm1

```diff
@@ -1402,98 +1403,189 @@ function DeActivate-AzureStack{
     } while ($currentAttempt -lt $maxAttempt)
 }
 
+function Get-AzToken
```

Comment:
should this go to separate file? that way all the places where needed can import function

Reply:
Agree. @bganapa any suggestion?

Reply:
Maybe here? Identity/AzureStack.Identity.psm1

Reply:
need this in a common location so that identity and registration script could share

Reply:
So maybe just make it part of the AzureStack module rather than keep it here?